shared/templates/create_audit_rules_dac_modification.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_dac_modification.py
+#        generate template-based checks for audit rules dac modifications
+
+
+import re
+
+from template_common import FilesGenerator, UnknownTargetError
+
+
+class AuditRulesDacModificationGenerator(FilesGenerator):
+    def __init__(self):
+        super(AuditRulesDacModificationGenerator, self).__init__()
+        self.delimiter = '&'
```

Comment:
this whole thing is not needed anymore, I would keep the ability to have different separators, that's a cool feature and we might use it in the future. but in audit rules we don't need it